src/ServiceFabric/ServiceFabric/Commands/AddAzureRmServiceFabricApplicationCertificate.cs

```diff
@@ -21,9 +21,11 @@
 using Microsoft.Azure.Commands.ServiceFabric.Models;
 using ServiceFabricProperties = Microsoft.Azure.Commands.ServiceFabric.Properties;
 using Microsoft.Azure.Commands.Common.Compute.Version_2018_04;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.ServiceFabric.Commands
 {
+    [CmdletDeprecation("This command will be deprecated. Please use Add-AzVmssSecret to add application certificates going forward.")]
```

Comment:
Please don't use `CmdletDeprecation` like this. The actual user experience may not be as you imagine -- 
![Annotation 2019-08-14 111938](https://user-images.githubusercontent.com/11371776/62992531-6885b900-be86-11e9-8f6b-22b5b4b9c6e7.png)

Instead, take the following example:
```cs
    [CmdletDeprecation(ReplacementCmdletName = VerbsCommon.Set +
        "-" +
        ResourceManager.Common.AzureRMConstants.AzureRMPrefix +
        DefinitionsCommon.DatabaseAuditCmdletsSuffix)]
```

Reply:
fixed, thanks